RuntimeGraphicsInfo.Plugin/RuntimeGraphicsInfo.cpp

```diff
@@ -60,6 +64,8 @@ OnGraphicsDeviceEvent(UnityGfxDeviceEventType eventType)
     case UnityGfxDeviceEventType::kUnityGfxDeviceEventInitialize:
     {
         s_RendererType = s_Graphics->GetRenderer();
+        if (s_RendererType == UnityGfxRenderer::kUnityGfxRendererMetal)
+            InitMetal(s_UnityInterfaces);
```

Comment:
why not just pass it in to the stats function like GetDeviceStatsD3D11?

Reply:
It actually have to initialize and cache some state and it doesn't seems semantically correct to make this initialization in Get method.